libs/client/cornflow_client/constants.py

```diff
@@ -107,6 +107,101 @@
+    ('time_limit', 'pulp', 'cbc') : 'timeLimit' ,
+    ('threads', 'pulp', 'cbc') : 'threads' ,
+    ('presolve', 'pulp', 'cbc') : 'presolve' ,
+    ('rel_gap', 'pulp', 'cbc') : 'gapRel' ,
+    
+
+    ('bar_iter_limit', 'pyomo', 'gurobi') : 'BarIterLimit' ,
+    ('best_obj_stop', 'pyomo', 'gurobi') : 'BestObjStop' ,
+    ('cutoff', 'pyomo', 'gurobi') : 'Cutoff' ,
+    ('iteration_limit', 'pyomo', 'gurobi') : 'IterationLimit' ,
+    ('mem_limit', 'pyomo', 'gurobi') : 'MemLimit' ,
+    ('solution_limit', 'pyomo', 'gurobi') : 'SolutionLimit' ,
+    ('time_limit', 'pyomo', 'gurobi') : 'TimeLimit' ,
+    ('abs_gap', 'pyomo', 'gurobi') : 'MIPGapAbs' ,
+    ('rel_gap', 'pyomo', 'gurobi') : 'MIPGap' ,
+    ('branch_dir', 'pyomo', 'gurobi') : 'BranchDir' ,
+    ('pump_passes', 'pyomo', 'gurobi') : 'PumpPasses' ,
+    ('heuristics', 'pyomo', 'gurobi') : 'Heuristics' ,
+    ('presolve', 'pyomo', 'gurobi') : 'Presolve' ,
+    ('threads', 'pyomo', 'gurobi') : 'threads' ,
+    
+    ('is_mip', 'pulp', 'gurobi') : 'mip' ,
+    ('time_limit', 'pulp', 'gurobi') : 'timeLimit' ,
+    ('presolve', 'pulp', 'gurobi') : 'presolve' ,
+    ('abs_gap', 'pulp', 'gurobi') : 'gapAbs' ,
+    ('rel_gap', 'pulp', 'gurobi') : 'gapRel' ,
+    ('feasibility_tol', 'pulp', 'gurobi') : 'FeasibilityTol' ,
+    ('iteration_limit', 'pulp', 'gurobi') : 'IterationLimit' ,
+
+
+    ('bar_tol', 'pyomo', 'scip') : 'numerics/barrierconvtol' ,
+    ('cutoff_breaker', 'pyomo', 'scip') : 'heuristics/shiftandpropagate/cutoffbreaker' ,
+    ('iteration_limit', 'pyomo', 'scip') : 'lp/iterlim' ,
+    ('mem_limit', 'pyomo', 'scip') : 'limits/memory' ,
+    ('solution_limit', 'pyomo', 'scip') : 'limits/maxsol' ,
+    ('time_limit', 'pyomo', 'scip') : 'limits/time' ,
+    ('rel_gap', 'pyomo', 'scip') : 'limits/gap' ,
+    ('pump_passes', 'pyomo', 'scip') : 'heuristics/feaspump/maxdepth' ,
+    ('presolve', 'pyomo', 'scip') : 'presolve' ,
+    ('threads', 'pyomo', 'scip') : 'threads' ,
+    ('nlp_tol', 'pyomo', 'scip') : 'heuristics/subnlp/opttol' ,
+    ('cutoff', 'pyomo', 'scip') : 'heuristics/subnlp/setcutoff' ,
+    ('nlp_iteration_limit', 'pyomo', 'scip') : 'heuristics/subnlp/itermin' ,
+    ('abs_gap', 'pyomo', 'scip') : 'limits/absgap ' ,
+
+    ('is_mip', 'pulp', 'scip') : 'mip' ,
+    ('abs_gap', 'pulp', 'scip') : 'gapAbs' ,
+    ('rel_gap', 'pulp', 'scip') : 'gapRel' ,
+    ('time_limit', 'pulp', 'scip') : 'timeLimit' ,
+    ('threads', 'pulp', 'scip') : 'threads' ,
+    ('max_nodes', 'pulp', 'scip') : 'maxNodes' ,
+    ('iteration_limit', 'pulp', 'scip') : 'lp/iterlim' ,
+    ('mem_limit', 'pulp', 'scip') : 'limits/memory' ,
+    ('cutoff_breaker', 'pyomo', 'scip') : 'heuristics/shiftandpropagate/cutoffbreaker' ,
+    ('iteration_limit', 'pyomo', 'scip') : 'lp/iterlim' ,
+    ('presolve', 'pyomo', 'scip') : 'presolve' ,
+    ('solution_limit', 'pyomo', 'scip') : 'limits/maxsol' ,
+    ('nlp_iteration_limit', 'pyomo', 'scip') : 'heuristics/subnlp/itermin' ,
```

Comment:
here there seems to be pyomo and pulp commands mixed together.
also iteration_limit is repeated.
I would advise putting the arguments always in the same order. That way, it will be easier to find them.

Reply:
Is this the last change? @AlejandroMus when do you think it will be done?

Reply:
Just done the push a few min ago. It is passing the unit tests